recipes/monsda/meta.yaml

```diff
@@ -22,15 +22,15 @@ build:
 
 requirements:
   host:
-    - python >=3.10
+    - python >=3.11.4
```

Comment:
You really need 3.11? Can you than specify please `3.11`

Reply:
Sure, will do